[2025-07-07T20:11:28.373+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: customer_activity_analysis.load_data manual__2025-07-07T20:11:14.759584+00:00 [queued]>
[2025-07-07T20:11:28.380+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: customer_activity_analysis.load_data manual__2025-07-07T20:11:14.759584+00:00 [queued]>
[2025-07-07T20:11:28.380+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2025-07-07T20:11:28.394+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): load_data> on 2025-07-07 20:11:14.759584+00:00
[2025-07-07T20:11:28.402+0000] {standard_task_runner.py:57} INFO - Started process 2190 to run task
[2025-07-07T20:11:28.404+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'customer_activity_analysis', 'load_data', 'manual__2025-07-07T20:11:14.759584+00:00', '--job-id', '93', '--raw', '--subdir', 'DAGS_FOLDER/customer_activity_dag.py', '--cfg-path', '/tmp/tmpqoktcqsu']
[2025-07-07T20:11:28.406+0000] {standard_task_runner.py:85} INFO - Job 93: Subtask load_data
[2025-07-07T20:11:28.418+0000] {logging_mixin.py:150} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-07-07T20:11:28.446+0000] {task_command.py:410} INFO - Running <TaskInstance: customer_activity_analysis.load_data manual__2025-07-07T20:11:14.759584+00:00 [running]> on host 7a17032e36f4
[2025-07-07T20:11:28.640+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='ds_const@mail.ru' AIRFLOW_CTX_DAG_OWNER='Daniil' AIRFLOW_CTX_DAG_ID='customer_activity_analysis' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2025-07-07T20:11:14.759584+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-07T20:11:14.759584+00:00'
[2025-07-07T20:11:29.206+0000] {logging_mixin.py:150} INFO - Successfully saved 220134 records to /opt/***/data/flags_activity.csv
[2025-07-07T20:11:29.209+0000] {logging_mixin.py:150} INFO - Файл /opt/***/shared_tmp/transform_manual__2025-07-07T20:11:14.759584+00:00.csv успешно удалён
[2025-07-07T20:11:29.209+0000] {logging_mixin.py:150} INFO - Temporary file /opt/***/shared_tmp/transform_manual__2025-07-07T20:11:14.759584+00:00.csv removed
[2025-07-07T20:11:29.216+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-07-07T20:11:29.224+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=customer_activity_analysis, task_id=load_data, execution_date=20250707T201114, start_date=20250707T201128, end_date=20250707T201129
[2025-07-07T20:11:29.257+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2025-07-07T20:11:29.273+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
